Dockerfile.template

```diff
@@ -1,4 +1,4 @@
 LABEL io.openshift.expose-services="3306:tcp" \
     io.openshift.tags="component,%NAME%"
-RUN apt-get update && apt-get install -y gettext-base python &&\
+RUN apt-get update && apt-get install -y gettext-base &&\
     cat /opt/app-root/etc/my.cnf > /etc/mysql/my.cnf
```

Comment:
This file could be replaced with `MYSQL_HOME` variable https://mariadb.com/kb/en/mariadb/configuring-mariadb-with-mycnf/